packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -192,6 +202,7 @@ function _Autocomplete<ItemType>(
     toggleMenu,
   } = useCombobox({
     items: flattenItems,
+    selectedItem,
```

Comment:
This changes the behavior of downshift since Autocomplete now becomes a "React Controlled Input" and downshift will stop managing the state on its own. After reading the [downshift docs on this topic](https://www.npmjs.com/package/downshift#control-props) in their README, I believe that it won't break other user cases. Still, we should make this clear by providing some explanation in the docs and add a small separate story in Storybook. Hope that's fine - I just want to make sure that we make users aware of this major change.

Reply:
@Lelith did you check whether we can utilize "initialSelectedItem" instead to avoid going down that road? It might save time and be saver but also, it might have the same shortcomings as "defaultValue" in our specific case.

Reply:
Yes, i added now more description to the new property. As its an optional one, it will not break existing cases. (i tested that) Also to avoid a breaking change i will not touch the defaultValue property, if someone is using this already.